Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -305,6 +312,15 @@ static void SendReply(const char* reply)
   }
 }
 
+static void WriteHostInfo()
+{
+  return SendReply(
+      StringFromFormat(
```

Comment:
Could we use fmt instead of StringFromFormat? The latter should be considered deprecated since we are migrating to fmt.

Reply:
I am not familiar with fmt, I just found this function when checking if we had a printf, how does that one work?

Reply:
`<fmt/format.h>` is already included so you just need to do `fmt::format("cputype:{};cpusubtype:{};...", MACH_O_POWERPC, MACH_O_POWERPC_750);`. It's similar to Python string formatting.